eox_nelp/processors/xapi/event_transformers/grade_events.py

```diff
@@ -67,3 +67,22 @@ def get_result(self):
                 "scaled": self.get_data("data.percent") or 0,
             }
         )
+
+    def get_context(self):
+        """
+        Get context for xAPI transformed event.
+        Returns:
+            `Context`
+        """
+        context = super().get_context()
+
+        attempts_extension = {
+            "http://id.tincanapi.com/extension/attempt-id": self.get_data("data.attempts")
```

Comment:
Here is possible to use the constants for these keys ?
https://github.com/nelc/event-routing-backends/blob/master/event_routing_backends/processors/xapi/constants.py#L51

Reply:
Great suggestion